io-engine/src/grpc/v1/replica.rs

```diff
@@ -121,7 +122,43 @@ impl ReplicaService {
         }
     }
 }
-
+fn filter_replicas_by_replica_type(
+    replicas: Vec<Replica>,
+    replica_type: i32,
```

Comment:
Use the actual type instead of i32, which can be retried with from_i32.

Reply:
fixed the code.